docs/open-source-project/porting/system.md

```diff
@@ -100,6 +100,14 @@ To use most of the `resourced` functionalities, you must enable the following `c
 >
 > To use the `resourced` freezer feature, you must install the freezer plugin by enabling `CONFIG_FREEZER`.
 
+`resourced` supports several swapping scenarios to provide more RAM to applications running in the foreground. To support these scenarios, enable the following options in the kernel configuration:
+
+  - `CONFIG_MEMCG_SWAP`: Adds swap management features to memory resource controller. Depends on `CONFIG_MEMCG` and `CONFIG_SWAP`.
+  - `CONFIG_ZRAM`: Creates memory-backed compressed block devices /dev/zramX (X = 0, 1, …).
+  - `CONFIG_ZRAM_LZ4_COMPRESS`: An alternative compression algorithm. By default, ZRAM uses Lempel–Ziv–Oberhumer (LZO).
```

Comment:
@surbhig12 In this list other items start with verb (Create, compress...etc), but for `CONFIG_ZRAM_LZ4_COMPRESS` only a noun(An alternative compression algorithm). Isn't it better to use same format for the items in a list?

Reply:
@mijin85cho,

I too had the same concern when I was reviewing this file. However, for this particular item, if I try following parallelism, it was changing the meaning.
 
Would you recommend changing to "Alternate compression algorithm" ?
(I understand, it still wouldn't solve the purpose)

Reply:
@steelman, thank you for your suggestions.

@mijin85cho , I too think both are okay to use. If I had to pick one, I would suggest using "Enables support for an alternative compression algorithm."

Regards,
Surbhi

Reply:
@surbhig12 Thanks for the opinion.

@steelman Would you please update with suggested sentence?

Reply:
Done.